.github/workflows/integration_test_charm.yaml

```diff
@@ -337,7 +337,18 @@ jobs:
         run: juju status --color --relations | tee ~/logs/juju-status.txt
       - name: juju debug-log
         if: ${{ success() || (failure() && steps.tests.outcome == 'failure') }}
-        run: juju debug-log --color --replay --no-tail | tee ~/logs/juju-debug-log.txt
+        # TODO future improvement: show models/ logs in juju-debug-log.txt to match `juju debug-log`
+        run: |
+          if [[ '${{ inputs.cloud }}' == 'lxd' ]]
+          then
+            mkdir ~/logs/models/
```

Comment:
```suggestion
            mkdir -p ~/logs/models/
```

Reply:
@carlcsaposs-canonical not if `failure() && steps.tests.outcome != 'failure'`